static/styles/left-sidebar.scss

```diff
@@ -172,12 +172,15 @@ li.active-sub-filter {
 }
 
 #global_filters .count,
-#stream_filters .count {
+#stream_filters .count,
+.topic-unread-count,
+.private_message_count{
     float: right;
-    line-height: 12px;
     background: hsl(105, 2%, 50%);
-    padding: 1px 4px;
-    color: hsl(0, 0%, 100%);
+    padding: 0 4px;
+    height: 15px;
+    line-height: 14px;
+    color: #fff;
```

Comment:
I think we want to use the `hsl` color style here (as we do in the rest of the app) so that we don't need an entry in `night_mode.css` for this style (at least, we attempted to purge use of colors specified this way from the codebase).  

Speaking of which, we should probably have a lint rule for that; I opened https://github.com/zulip/zulip/issues/9689 to track that issue.